_posts/2021-04-28-#17331.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: pr
+date: 2021-04-28
+title: "Use effective values throughout coin selection"
+pr: 17331
+authors: [achow101]
+components: ["wallet"]
+host: Xekyo
+status: upcoming
+commit: 4ac1add
+---
+
+## Notes
+
+- Wallets maintain their balance as a pool of Unspent Transaction Outputs
+  (UTXOs).
+
+- _Coin selection_ refers to the process of picking UTXOs from the wallet's UTXO
+  pool to fund a transaction.
+
+- What are the goals of coin selection? Economics, privacy, reliability of
+  inputs/conflicts
+
+- Why are we trying to not spend unconfirmed inputs?
+
+- When a new transaction is built, the target feerate is provided as an input.
+
+- A UTXOs _effective value_ is its value reduced by its cost to be spent at a
+  given feerate.
```

Comment:
Perhaps go into a bit more detail here. Why does an input have a 'cost to be spent at a given feerate'.

Reply:
Have elaborated this point